resources.whatwg.org/browser-logos/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This should probably go in the root, not deployed onto our resources.whatwg.org server in a browser-logos subdirectory. Same with the .sh script.

Reply:
Done.

---

Yes, that's what I was thinking in the beginning, but I wasn't sure if I should add it the root or in `resources.whatwg.org`. In the end I decided to not be intrusive and just place it where it was needed.